config/terraform/application/variables.tf

```diff
@@ -93,6 +93,10 @@ variable "postgres_flexible_server_sku" {
   default = "B_Standard_B1ms"
 }
 
+variable "postgres_snapshot_flexible_server_sku" {
+  default = "GP_Standard_D2ds_v4"
+}
```

Comment:
I think this might be a bit overpowered (and expensive) for the snapshot server, we should probably be using a burstable plan - and then upgrade it later if needed.

Can it be `B_Standard_B1ms` for now?

Reply:
Sure